tests/renderStatsCard.test.js

```diff
@@ -75,6 +75,22 @@ describe("Test renderStatsCard", () => {
     expect(queryByTestId(document.body, "rank-circle")).not.toBeInTheDocument();
   });
 
+  it("should render with custom width set", () => {
+    document.body.innerHTML = renderStatsCard(stats);
+    expect(document.querySelector("svg")).toHaveAttribute("width", "495");
+
+    document.body.innerHTML = renderStatsCard(stats, { card_width: 400 });
+    expect(document.querySelector("svg")).toHaveAttribute("width", "400");
+  });
+
+  it("should render with custom width set and limit minimum width", () => {
+    document.body.innerHTML = renderStatsCard(stats, { card_width: 1 });
+    expect(document.querySelector("svg")).toHaveAttribute("width", "340");
+
+    document.body.innerHTML = renderStatsCard(stats, { card_width: 1, hide_rank: true });
+    expect(document.querySelector("svg")).toHaveAttribute("width", "305.81250000000006");
```

Comment:
@postatum It looks like the current card width with this setting should be 270 px.

![Rick Staa's Github stats](https://github-readme-stats-taupe-pi.vercel.app/api?username=rickstaa&card_width=1&hide_rank=true)